### DISCUSSION TITLE
What's new in Astroid 2.6.1

### DISCUSSION THREAD

**Discussion by sonvnn** [templaza/astroid-framework#221] [Announcements]
### Finally! Child Template is here

In Joomla 4! Child template is available. Please check video to learn more why child template is necessary for Joomla template 
https://www.youtube.com/watch?v=tUDXqlmRZKg

### Left and Right Canvas Button Position

Go to template options > Header > Off-Canvas Menu
![Screen Shot 2022-10-11 at 16 24 09](https://user-images.githubusercontent.com/3204103/195052026-d92c0687-7157-427c-9dcb-61b2e0a4ea71.png)
![Screen Shot 2022-10-11 at 16 22 32](https://user-images.githubusercontent.com/3204103/195052057-56468401-8581-4ded-ae7e-00ecada6790f.png)
![Screen Shot 2022-10-11 at 16 22 45](https://user-images.githubusercontent.com/3204103/195232728-53cc2220-a761-4d01-87cd-ecfa75ae4e8a.png)

### Add more 4 Preload Styles

![Screen Shot 2022-10-11 at 16 25 51](https://user-images.githubusercontent.com/3204103/195052516-3c01a74f-ea1b-4a95-af1c-d99834bc8592.png)

### Enable Spectrum Palette

![Screen Shot 2022-10-11 at 16 26 46](https://user-images.githubusercontent.com/3204103/195052791-b1efe16f-3d17-4508-8866-99c014bb94b6.png)

**Comment by sonvnn** [comment]
Hi @Chacapamac @Joomlaplates 

Please check this video about Child-Template https://www.youtube.com/watch?v=tUDXqlmRZKg
We will prepare to running to Joomla 4.3 and Joomla 5

Thank you very much for cooperation
Best Regards,
Sonny

**Reply by Chacapamac** [reply]
Like your video
Good Day!
